packages/babel-plugin-transform-runtime/src/index.js

```diff
@@ -15,13 +17,11 @@ export default function({ types: t }) {
 
   return {
     pre(file) {
-      const moduleName = getRuntimeModuleName(this.opts);
+      const moduleName = getRuntimeModuleName(options);
 
-      if (this.opts.helpers !== false) {
-        const baseHelpersDir = this.opts.useBuiltIns
-          ? "helpers/builtin"
-          : "helpers";
-        const helpersDir = this.opts.useESModules
+      if (helpers !== false) {
+        const baseHelpersDir = useBuiltIns ? "helpers/builtin" : "helpers";
```

Comment:
This and the next few lines are all validation for the options, so that should get moved up too.